### ISSUE TITLE
Hi.

### ISSUE THREAD

**Issue by leegod** [TheSniperFan/unityserializer-ng#35]
So I need save game system that can save entire scene's current state.  

At past I used this asset,  

But this stops support and does not compatible with unity5.0~ 
So does this work at unity newest version? 5.4

If not work, do you know any other solutions?

Thanks.

**Comment by TheSniperFan** [comment]
At the project's new home [right here](https://gitgud.io/TheSniperFan/unityserializer-ng) you can download the master branch, which is made for **Unity 5.4 only**.
Please bear in mind that Unity did some quite substantial changes in 5.2 and 5.3 that break backwards compatibility to older versions (new version of PhysX and the scene manager). If you upgrade an existing project, be sure to test thoroughly.

**Comment by leegod** [comment]
p {margin-top:0px;margin-bottom:0px;} 

Thanks!So this is only solution to save entire scene in one click, right?--------- 원본 메일 ---------보낸사람: TheSniperFan notifications@github.com받는사람 : TheSniperFan/unityserializer-ng unityserializer-ng@noreply.github.com참조 : leegod wsxdrfv@hanmail.net,Author author@noreply.github.com날짜: 2016년 9월 05일 월요일, 18시 55분 24초 +0900제목: Re: [TheSniperFan/unityserializer-ng] Hi. (#35)

```
At the project's new home right here you can download the master branch, which is made for Unity 5.4 only.
```

Please bear in mind that Unity did some quite substantial changes in 5.2 and 5.3 that break backwards compatibility to older versions (new version of PhysX and the scene manager). If you upgrade an existing project, be sure to test thoroughly.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

p  {font-size:11pt ! important;font-family:verdana,'굴림',gulim,tahoma,sans-serif ! important;}

**Comment by TheSniperFan** [comment]
I don't know, if there are others. I used this one for my project before I started porting it to Unreal 4.

**Comment by leegod** [comment]
p {margin-top:0px;margin-bottom:0px;} 

Anyway, thanks, So this save system's usage is same with before? Attach component to object that want to be saved, and make SaveGameManager object?--------- 원본 메일 ---------보낸사람: TheSniperFan notifications@github.com받는사람 : TheSniperFan/unityserializer-ng unityserializer-ng@noreply.github.com참조 : leegod wsxdrfv@hanmail.net,Author author@noreply.github.com날짜: 2016년 9월 05일 월요일, 20시 37분 59초 +0900제목: Re: [TheSniperFan/unityserializer-ng] Hi. (#35)

```
I don't know, if there are others. I used this one for my project before I started porting it to Unreal 4.
```

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

p  {font-size:11pt ! important;font-family:verdana,'굴림',gulim,tahoma,sans-serif ! important;}

**Comment by TheSniperFan** [comment]
Basically the same. Some changes were made to how materials are handled, but I will start documenting these soon.

**Comment by leegod** [comment]
p {margin-top:0px;margin-bottom:0px;} 

Hi. Thx.But how to install this to existing unity5.4 project?Just copy whole folders(Plugins and UnitySerializerExamples) to existing project and start using?--------- 원본 메일 ---------보낸사람: TheSniperFan notifications@github.com받는사람 : TheSniperFan/unityserializer-ng unityserializer-ng@noreply.github.com참조 : leegod wsxdrfv@hanmail.net,Author author@noreply.github.com날짜: 2016년 9월 05일 월요일, 21시 16분 41초 +0900제목: Re: [TheSniperFan/unityserializer-ng] Hi. (#35)

```
Basically the same. Some changes were made to how materials are handled, but I will start documenting these soon.
```

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

p  {font-size:11pt ! important;font-family:verdana,'굴림',gulim,tahoma,sans-serif ! important;}

**Comment by TheSniperFan** [comment]
Yes. You don't need the examples though, if you don't want them. For a quick start it's a good idea to include them, because they have a simple pause menu in which you can save/load.
